### PR TITLE
[ez][2/n][AIC-py] rm unused imports

### PR DESCRIPTION
**Pull request by jonathanlastmileai**
[ez][2/n][AIC-py] rm unused imports

### CODE REVIEW

File: python/tests/test_run_config.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from aiconfig.Config import AIConfigRuntime
 from mock import patch
 
-from .conftest import mock_openai_chat_completion, set_temporary_env_vars
+from .conftest import mock_openai_chat_completion
```
Comment (rholinshead) [inline python/tests/test_run_config.py:6]: Just confirming it's not needed in the function def below? Not sure if the pytest decorator does magic like calling with this as the param

File: python/src/aiconfig/util/config_utils.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
-    from aiconfig.Config import AIConfigRuntime
+    pass
```
Comment (rholinshead) [inline python/src/aiconfig/util/config_utils.py:6]: Should this just be removed and move the InferenceSettings import up?

**Review by rholinshead — APPROVED** [review]
Accepting to unblock, just minor things

File: python/tests/test_run_config.py (hunk 1 above)
Comment (jonathanlastmileai) [reply python/tests/test_run_config.py:6]: No magic AFAIK. test passes

**Pull request merged**